lib/recurly/hosted-fields.js

```diff
@@ -29,13 +30,55 @@ export class HostedFields extends Emitter {
     this.errors = [];
     this.initQueue = [];
     this.readyState = 0;
+    this.onTabNext = this.onTabNext.bind(this)
+    this.onTabPrevious = this.onTabPrevious.bind(this)
     this.configure(options);
     this.inject();
     this.on('hostedField:state:change', this.update.bind(this));
     this.on('bus:added', bus => {
       this.bus = bus;
       this.fields.forEach(hf => bus.add(hf));
     });
+
+    this.on('hostedField:tab:previous',this.onTabPrevious)
+    this.on('hostedField:tab:next',this.onTabNext)
+  }
+
+  onTabNext(body) {
+    let type = body.type
+    let field = this.fields.find(function(f){ return f.type === type })
+    let allFields = document.querySelectorAll("[data-recurly]")
```

Comment:
So the one issue this may present is if the fields are not bound using this selector. It's possible for the merchant to specify other selectors. I think we'll need to incorporate the fields known by the `HostedFields` instance.